api/src/shared/infrastructure/repositories/user-repository.js

```diff
@@ -295,32 +280,29 @@ const updateHasSeenLevelSevenInfoToTrue = async function (id) {
 };
 
 const updateHasSeenNewDashboardInfoToTrue = async function (id) {
-  const user = await BookshelfUser.where({ id }).fetch({ require: false });
-  await user.save({ hasSeenNewDashboardInfo: true }, { patch: true, method: 'update' });
-  return bookshelfToDomainConverter.buildDomainObject(BookshelfUser, user);
+  const [user] = await knex('users').where({ id }).update('hasSeenNewDashboardInfo', true).returning('*');
```

Comment:
Bookshelf pouvait se charger de modifier automatiquement le `updatedAt` d'un enregistrement, mais avec Knex c'est à faire manuellement à chaque requête effectuant une modification :

```suggestion
  const [user] = await knex('users').where({ id }).update({
    hasSeenNewDashboardInfo: true,
    updatedAt: new Date(),
  }).returning('*');
```